server/lib/nl/detection/detector.py

```diff
@@ -65,13 +65,13 @@ def detect(
   #
   # In the absence of the PALM API key, fallback to heuristic.
   #
-  if (detector_type in _PALM_API_DETECTORS and
-      'PALM_API_KEY' not in current_app.config):
+  if (detector_type in _LLM_API_DETECTORS and
+      'LLM_API_KEY' not in current_app.config):
     counters.err('failed_palm_keynotfound', '')
```

Comment:
should we update this to 'failed_llm_keynotfound'?

Reply:
yes!